src/functions/times.ts

```diff
@@ -0,0 +1,83 @@
+import { SolutionBuilder } from '../solution';
+
+export type definition = (times: number, data: any) => any | undefined;
+
+export const solution = new SolutionBuilder<definition>('times');
+
+/**
+ * Solution Definition
+ */
+
+solution
+  .description(
+    'times receives as arguments the number of iterations and a function to execute n times and returns an array of the results. Very useful when creating dynamic test data.'
+  )
+  .example('r.times(2, null) // [null, null]')
+  .example('r.times() // []')
+  .example('r.times(2) // [undefined, undefined]')
```

Comment:
`.test('Return empty array', [], [])` this tests does not applied as described. We have to double-check our code base for this.